### PR TITLE
Adds Klingoffs and Space Elves to the game as roundstart species

### PR DESCRIPTION
**Pull request by Iamgoofball**
🆑 Iamgoofball and some spriter i forgot their name
add: Added two new alien races to the game, Klingoffs and Space Elves!
/🆑

## **Klingoffs**

A species of warriors, locked in an endless civil war on their home planet over putting butter on toast butter side up or butter side down. Nanotrasen often hires them as security forces due to their hostile nature and lack of reluctance to put boots to asses during riots.

## **Space Elves**

Space Elves have been floating around the galaxy for ages, spreading their peace-and-love-unless-you-hurt-trees message, but Nanotrasen only just finally convinced the Space Elves that towercaps are not trees and therefore it is safe to chop them down. Space Elves are universally hated by everyone for being Space Elves. Fuck Space Elves.

These two races are:
1. Unique sprites
2. Not just "X animal but anthro"
3. Actually have some backstory
4. Will create fun and interesting roleplay situations

### CODE REVIEW

**Comment by Armhulen** [comment]
better than humans with cat parts

**Comment by MoreRobustThanYou** [comment]
i'd review this but i have a headache

File: code/modules/mob/living/carbon/human/species_types/klingoffs_elves.dm [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/datum/species/klingoff
+	name = "Klingoff"
+	id = "klingoff"
+	default_color = "FFFFFF"
+	specflags = list(EYECOLOR,HAIR,FACEHAIR,LIPS)
+	mutant_bodyparts = list()
+	default_features = list()
+
+/datum/species/space_elf
```
Comment (KorPhaeron) [inline code/modules/mob/living/carbon/human/species_types/klingoffs_elves.dm]: These should be mutant parts if they're just ears

**Review by imtakingabreakdontatme — CHANGES_REQUESTED** [review]

File: config/game_options.txt [hunk 2]
```diff
@@ -394,6 +394,8 @@ JOIN_WITH_MUTANT_RACE
 
 ## You probably want humans on your space station, but technically speaking you can turn them off without any ill effect
 ROUNDSTART_RACES human
+ROUNDSTART_RACES space_elf
+ROUNDSTART_RACES klingoff
```
Comment (DoctoreMad) [inline config/game_options.txt]: Not humans. But let me guess this is here because they're the same as humans?

Comment (Iamgoofball) [reply config/game_options.txt]: mhm

File: code/modules/mob/living/carbon/human/species_types/klingoffs_elves.dm (hunk 1 above)
Comment (Iamgoofball) [reply code/modules/mob/living/carbon/human/species_types/klingoffs_elves.dm]: Reasons this is dumb:

I'm sacrificing the unique species aspect
I cant make them forced vegetarian
I have to hardcode mutant part checks for shit with elves vs catbeest retards

Comment (Iamgoofball) [reply code/modules/mob/living/carbon/human/species_types/klingoffs_elves.dm]: before you say "make the ears do the effects"

1. why would I want ears to make you vegetarian
2. i refuse to allow elves with cat tails

**Comment by PopNotes** [comment]
-1 fuck elves

**Comment by Iamgoofball** [comment]
@popnotes with this PR you can genocide elves

File: code/modules/mob/living/carbon/human/species_types/klingoffs_elves.dm (hunk 1 above)
Comment (Supermichael777) [reply code/modules/mob/living/carbon/human/species_types/klingoffs_elves.dm]: 1. surprisingly principled of you considering how you normally work.
2. we can actually do that with surgery

**Comment by DoctoreMad** [comment]
*Only if you're an antag

**Comment by PopNotes** [comment]
🤔

**Comment by KorPhaeron** [comment]
Basically everything that is wrong with catgirls (no interesting mechanics, hard to distinguish from humans, not very original) is wrong with these and I get the feeling the PR exists to bait me into either merging bloat or appearing a hypocrite. 

I guess I'll unsubscribe and check back in a week to read the 400 comments of arguing about cat ears

**Comment by ChangelingRain** [comment]
I'll just solve kor's dilemma by closing it myself, since then he doesn't have to deal with it without taking a stance himself.

**Comment by Iamgoofball** [comment]
I refuse to respect your maintainer status after literally sabotaging pull requests for a server you're permabanned from.

**Comment by Iamgoofball** [comment]
https://tgstation13.org/phpBB/viewtopic.php?f=23&t=13010 never forget

**Comment by Doppeltongue** [comment]
I am angry. ANGRY ABOUT ELVES!

**Comment by Dorsisdwarf** [comment]
I like klingoffs, but they should be role-restricted exclusively to security. Space elves only get hydro and assistant because they're filthy hippies or dirty vagrants.

**Comment by Bawhoppen** [comment]
Doesn't seem like anything unique for them though. I understand not disturbing balance but maybe make them more distinct in some way? (Catgirls are distinct and unique)

Def make klingoffs only security though if nothing else, and probably limit space elves in some way too

**Comment by Iamgoofball** [comment]
Please explain how catgirls are distinct and unique gameplay wise

**Comment by Bawhoppen** [comment]
Who ever said anything about gameplay?
(Though these do look a little too similar to humans maybe on first glance)

**Comment by Iamgoofball** [comment]
You mentioned balance so gameplay is implied. C- , see me after preschool

**Comment by Bawhoppen** [comment]
I said I understand you shouldn't disturb balance. AKA don't do anything to them mechanically disruptie (which includes potentially anything at all)

I'm talking about their flavor and associated presence in culture and such. These seem like just little spoofs of some tropes, while catgirls for example have alot of ubiquity and a special place in our culture.

**Comment by kevinz000** [comment]
fucking klingoffs taking the jobs from the one true catcurity officer :-1:

**Comment by CPTANT** [comment]
Seems rather hard for the AI to actually distinguish them from humans.

**Comment by FantasticFwoosh** [comment]
@KorPhaeron stop projecting , these two races are less collectively cancerous than catpeople who make it a goal to be as visible and obnoxious as possible and from the outset these races are distinct alien/human mutations.

- Catpeople lore is ambigiously revolving around pseudo beastiality, and nobody has actually decided on concrete catpeople lore publically other than they are abominations and forced Neoparika visual novel memes. These new races are just from space and have RP suggested behaviour patterns, this is leagues better for the game than 'nya cat cafe, and having promiscous catpeople players constantly force themselves upon you in the dorms.'

Also they're a clean slate compared to catpeople who've had features added in small & minor ways that essentially make them impure from the claim they are only sprited mutants since it alters the humans they are based off. **Essentially these two races are just REAL sprited mutants and attempts to change that defacto-post merge should probably be veto'd**.

**Comment by InsaneHyena** [comment]
This is amazing, merge immediately.

**Comment by Dorsisdwarf** [comment]
I mean theres a very good point that these races are *even harder* for silicons to tell from humans than dirty cats

**Comment by CPTANT** [comment]
I was already thinking: "Would I consider these as non human if I saw them"

The difference is literally 2 pixels. The klingoff just looks like a bald human and the elf is pretty hard to distinguish from someone with a wacky hairstyle.

**Comment by DoctoreMad** [comment]
Just so I can remember, standard policy is for the AI to assume anyone they don't know the race of is human right?

**Comment by nicbn** [comment]
Close this. Make it better or don't make it.

**Comment by nicbn** [comment]
Also no Klingoff language? This is not even a real PR.

**Comment by ChangelingRain** [comment]
here is a kor comment
>Basically everything that is wrong with catgirls (no interesting mechanics, hard to distinguish from humans, not very original) is wrong with these and I get the feeling the PR exists to bait me into either merging bloat or appearing a hypocrite.

accordingly, here is a solution

**Comment by The-clown-known-as-Pepe** [comment]
24 hour rule?

**Comment by Bawhoppen** [comment]
This has 9 thumbs up and no thumbs down and the 24hour rule. Reopen it

**Comment by KorPhaeron** [comment]
I hope whoever popularized "you can't reject a PR for an arbitrary 24 hour period" feels guilt for what they've done

**Comment by Bawhoppen** [comment]
how is 24 hours arbitrary?

**Comment by Armhulen** [comment]
^^

though yeah these guys are boring as hell

**Comment by Iamgoofball** [comment]
I need to update the demo'd sprite

Also, we didn't have languages back when these were coded and I'll gladly add some.

**Comment by The-clown-known-as-Pepe** [comment]
@Iamgoofball if you can make their special features go thru most of the clothing that would be rad for silicons

**Comment by Iamgoofball** [comment]
Also, those concerned about species checking, names and medical records exist

**Comment by Iamgoofball** [comment]
Additionally I got updated klingoff sprites soon

**Comment by FantasticFwoosh** [comment]
Klingoffs can speak draconic, and elves can speak slime seems like a language compromise if you are REALLY pressing to give them a unique feature (though i sort of wish you didn't)

**Comment by Mey-Ha-Zah** [comment]
There is always the option of giving the AI and borgs a genetic facial detector. Which would show in the corner of the character like a sec officer's Huds if they are human or not while not wearing anything hiding their face. If their face and or other features are hidden, then it shows a blank and AI has to account for information other than features if human or not (Current ID, voice, ect.)

**Comment by Iamgoofball** [comment]
No fwoosh they're going to speak elvish and klingoff

**Comment by kevinz000** [comment]
fwoosh being dumb per usual lmao

**Comment by DoctoreMad** [comment]
>More unique languages for some shoot-off species that barely anyone even uses in-game

WHO CARES

**Comment by optimumtact** [comment]
no new roundstart races

**Comment by optimumtact** [comment]
oh never mind this has 6 more hours

**Comment by DoctoreMad** [comment]
>no new roundstart races

Huh.

**Comment by optimumtact** [comment]
Yes that is my design direction for the game

**Comment by TheDreamweaver** [comment]
I'm just here to maintain the image that I'm still involved in /tg/ coding.

**Comment by banthebantz** [comment]
new races that aren't furshit.

good

**Comment by The-clown-known-as-Pepe** [comment]
when did we get furshit races? 🤔

**Comment by lzimann** [comment]
your avatar is a good example

**Comment by Doppeltongue** [comment]
Any last words before the 24 hours are up? Definitely seems like the sprites are too close to humans for silicons to handle. At least catpeople have obvious tails (at round start at least) for the silicons to use for discrimination.

**Comment by optimumtact** [comment]
@GunHog your dissent is noted and I will punish your chosen roles harshly

**Comment by lzimann** [comment]
lol

**Comment by Bawhoppen** [comment]
>14 thumbs up
>2 thumbs down

>gets closed 

Out of touch?
:thinking:

**Comment by lzimann** [comment]
\>using popularity argument

**Comment by Bawhoppen** [comment]
>not caring about popularity

**Comment by optimumtact** [comment]
\>not escaping your meme pointers

**Comment by Iamgoofball** [comment]
@optimumtact Can I remove the existing 2 aliens and replace them with these? That way it's replacing instead of adding new ones.

**Comment by optimumtact** [comment]
you're funny, cross my design vision again and I'll gitban you

**Comment by Iamgoofball** [comment]
you do not hold the conch citrus man

**Comment by banthebantz** [comment]
citrus man confirmed to prefer FURRY RACES to REAL RACES

**Comment by TheDreamweaver** [comment]
https://i.redd.it/e4lc37hcafpz.jpg

**Comment by MisterLayne** [comment]
not reopened yet

**Comment by optimumtact** [comment]
cross my design vision and I"ll gitban you

**Pull request closed without merging**